pom.xml

```diff
@@ -1530,7 +1524,7 @@
             <exclude>${stress.skip.pattern}</exclude>
             <exclude>${test.exclude.pattern}</exclude>
           </excludes>
-          <argLine>-Xmx${test.maxHeapSize} ${jvm.opts} -Dorg.geotools.test.extensive=${extensive.tests} -Dorg.geotools.test.interactive=${interactive.tests} -Dorg.geotools.image.test.skip=${skip.image.tests} -Dorg.geotools.image.test.interactive=${interactive.image} -Djava.awt.headless=${java.awt.headless} -Dsun.java2d.d3d=${sun.java2d.d3d} -Djava.io.tmpdir="${java.io.tmpdir}" -Djava.library.path="${java.library.path}" ${test.args}  -XX:+TieredCompilation -XX:TieredStopAtLevel=1 -XX:+IgnoreUnrecognizedVMOptions --add-modules=java.xml.bind --add-modules=java.activation</argLine>
+          <argLine>-Xmx${test.maxHeapSize} ${jvm.opts} -Dorg.geotools.test.extensive=${extensive.tests} -Dorg.geotools.test.interactive=${interactive.tests} -Dorg.geotools.image.test.skip=${skip.image.tests} -Dorg.geotools.image.test.interactive=${interactive.image} -Djava.awt.headless=${java.awt.headless} -Dsun.java2d.d3d=${sun.java2d.d3d} -Djava.io.tmpdir=${java.io.tmpdir} -Djava.library.path="${java.library.path}" ${test.args}  -XX:+TieredCompilation -XX:TieredStopAtLevel=1 -XX:+IgnoreUnrecognizedVMOptions --add-modules=java.xml.bind --add-modules=java.activation</argLine>
```

Comment:
@vjeetje please explain or revert the `argLine` `tmpdir` quote removal.

Reply:
@bencaradocdavies This is some weird Intellij interaction with maven-surefire-plugin. I'm not sure about the cause but running a test on Intellij with -Djava.io.tmpdir="${java.io.tmpdir}" -Djava.library.path="${java.library.path}" fails with a parsing problem. Removing the quotes around tmpdir fixes those issues. The build in general works fine though without my changes. I'll revert those changes as I can't find a solution that works in general on all systems with the IDE.  I've added a screenshot showing the work-around for those on Windows using Intellij. Just uncheck the argline checkbox.
<img width="614" alt="argline_intellij" src="https://user-images.githubusercontent.com/1086527/40968373-b11f0342-68b5-11e8-9201-7a67a51b4e50.png">